html-aam/accname-computation-by-element/section-and-grouping-elements.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname computation tests for section and grouping elements</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <script src="/html-aam/scripts/html-aam-accname-utils.js"></script>
+</head>
+
+<body>
+  <script>
+    const testConfig = {
+      urlSpec: 'https://www.w3.org/TR/html-aam-1.0/#section-and-grouping-element-accessible-name-computation',
+      elements: {
+        'header': {},
+        'nav': {},
+        'search': {},
+        'form': {},
+        'main': {},
+        'section': {},
```

Comment:
Interestingly, a `<section>` without an accessible name is treated as a generic container, so "to be empty" might not apply in that context. That said, it may not be worth creating two separate test cases.